frontend/src/UserData.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useJwt } from "react-jwt";
+import useToken from './hooks/useToken'
+
+function UserData() {
+  const { token } = useToken();
+  const { decodedToken } = useJwt(token);
+  if (decodedToken) {
+    return (
+      <>
+        <div>{decodedToken.name}</div>
+        <div>{decodedToken.email}</div>
+      </>
+    );
+  }
+  else {
+    return <div></div>
```

Comment:
Erre az empty  '<div></div>' -re szükség van? nem lehet csak '' string a return? 🤔

Reply:
Javítottam:
return (
    <>
      <div>{decodedToken?.name || ''}</div>
      <div>{decodedToken?.email || ''}</div>
    </>
  );
Így még jobb is, mert nem száll el, ha mondjuk a decodedToken létezik, de valamiért nincs benne name vagy email.